modules/ROOT/pages/testing-writing-your-first-munit-test.adoc

```diff
@@ -0,0 +1,201 @@
+                 <phase>integration-test</phase>
+             </execution>
+         </executions>
+        <dependencies>
+             <dependency>
+                 <groupId>com.mulesoft.munit</groupId>
+                 <artifactId>munit-runner</artifactId>
+                 <version>${munit.version}</version>
+                 <classifier>mule-plugin</classifier>
+             </dependency>
+             <dependency>
+                 <groupId>com.mulesoft.munit</groupId>
+                 <artifactId>munit-tools</artifactId>
+                 <version>${munit.version}</version>
+                 <classifier>mule-plugin</classifier>
+             </dependency>
+         </dependencies>
+      </plugin>
+      ...
+    </plugins>
+  </build>
+  ...
+</project>
+----
+
+[[write-your-test]]
+== Write Your MUnit Test
+
+To test your module, your MUnit test must execute the module's operations and make assertions about the result of these executions.
+
+The MUnit test XML must be placed inside the `src/test/munit` folder of the module project.
+
+[source,xml,linenums]
+----
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:basic="http://www.mulesoft.org/schema/mule/basic"
+      xmlns:munit="http://www.mulesoft.org/schema/mule/munit"
+      xmlns:munit-tools="http://www.mulesoft.org/schema/mule/munit-tools"
+      xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+      http://www.mulesoft.org/schema/mule/munit-tools http://www.mulesoft.org/schema/mule/munit-tools/current/mule-munit-tools.xsd
+      http://www.mulesoft.org/schema/mule/munit http://www.mulesoft.org/schema/mule/munit/current/mule-munit.xsd
+          http://www.mulesoft.org/schema/mule/basic http://www.mulesoft.org/schema/mule/basic/current/mule-basic.xsd">
+
+    <basic:config name="config" configId="configId">
+        <basic:connection requiredParameter="aValue" />
+    </basic:config>
+
+    <munit:config name="basic-operations-test"/>
+
+    <munit:test name="sayHiTest">
+        <munit:execution>
+            <basic:say-hi person="Your Name"/>
+        </munit:execution>
+        <munit:validation>
+            <munit-tools:assert-equals actual="#[payload]" expected="#['Hello!']"/>
+        </munit:validation>
+    </munit:test>
+
+    <munit:test name="retrieveInfoTest">
+        <munit:execution>
+            <basic:retrieve-info config-ref="config"/>
+        </munit:execution>
+        <munit:validation>
+            <munit-tools:assert-equals actual="#[payload]" expected="#['Using Configuration [configId] with Connection id [aValue:100]']"/>
+        </munit:validation>
+    </munit:test>
+
+</mule>
+----
+
+[[run-your-test]]
+== Run Your MUnit Test
+
+Run your test from the command line using Maven:
+
+[source,bash,linenums]
+----
+mvn clean verify
+----
+
+This outputs the result of your test run:
+----
```

Comment:
There was a discussion recently about what attributes to give to CL output. I'm not sure what we decided. bash maybe?

Reply:
just looked at the thread, and yup bash works